release-notes/opensearch.release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,415 @@
+## Version 1.0.0-rc1 Release Notes
+
+* __[BWC] fix mixedCluster and rolling upgrades (#775) (#793)__
+
+    [Nick Knize](mailto:nknize@gmail.com) - Fri, 28 May 2021 10:08:05 -0500
+
+    EAD -&gt; refs/heads/1.0, tag: refs/tags/1.0.0-rc1, refs/remotes/origin/1.0
+    This commit fixes mixedCluster and rolling upgrades by spoofing OpenSearch
+
+    version 1.0.0 as Legacy version 7.10.2. With this commit an OpenSearch 1.x node
+     can join a legacy (&lt;= 7.10.2) cluster and rolling upgrades work as expected.
+
+    Mixed clusters will not work beyond the duration of the upgrade since shards
+
+    cannot be replicated from upgraded nodes to nodes running older versions.
+     Signed-off-by: Nicholas Walter Knize &lt;nknize@apache.org&gt;
+     Co-authored-by: Shweta Thareja &lt;tharejas@amazon.com&gt;
+
+
+* __[BUG] fix MainResponse to spoof version number for legacy clients (#708)__
+
+    [Nick Knize](mailto:nknize@gmail.com) - Fri, 28 May 2021 10:07:06 -0500
+
+
+    This commit changes MainResponse to spoof OpenSearch 1.x version numbers as
+
+    Legacy version number 7.10.2 for legacy clients.
+     Signed-off-by: Nicholas Walter Knize &lt;nknize@apache.org&gt;
+
+* __[CVE] Upgrade dependencies for Azure related plugins to mitigate CVEs (#688) (#771) (#784)__
+
+    [Abbas Hussain](mailto:abbas_10690@yahoo.com) - Thu, 27 May 2021 00:43:40 +0530
+
+
+    * Update commons-io-2.4.jar to 2.7 for plugins/discovery-azure-classic module
+    * Remove unused jackson dependency and respective LICENSE and NOTICE
+    * Update guava dependency to mitigate CVE for repository-azure plugin
+     Signed-off-by: Abbas Hussain &lt;abbas_10690@yahoo.com&gt;
+
+* __distribution/packages: Fix RPM architecture name for 64-bit x86 (#620) (#770) (#783)__
+
+    [Abbas Hussain](mailto:abbas_10690@yahoo.com) - Wed, 26 May 2021 23:55:35 +0530
+
+
+    RPM uses the &#34;x86_64&#34; name for 64-bit x86, which is in-line with GCC
```

Comment:
This commit doesn't mention the author. Is `Signed-off ` by enough?

Reply:
Yeah definitely echo your sentiments.
I would ideally love something like: 
Category:
PR title (#number)
PR description
Authored by abc@

Lets work on this for the next release. As we have discussed we have to diligent in our back port PRs so that authors are given the credit.
Issue: #789